<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Water</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="title">
       <string>Ripple Controls</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>17</x>
         <y>25</y>
         <width>191</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Amplitude</string>
       </property>
      </widget>
      <widget class="QSlider" name="speedSlider">
       <property name="geometry">
        <rect>
         <x>17</x>
         <y>95</y>
         <width>131</width>
         <height>19</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="energySlider">
       <property name="geometry">
        <rect>
         <x>17</x>
         <y>145</y>
         <width>131</width>
         <height>19</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>17</x>
         <y>75</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Speed</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>17</x>
         <y>125</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Energy</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="amplitudeDisplay">
       <property name="geometry">
        <rect>
         <x>162</x>
         <y>45</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="speedDisplay">
       <property name="geometry">
        <rect>
         <x>162</x>
         <y>95</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="energyDisplay">
       <property name="geometry">
        <rect>
         <x>162</x>
         <y>145</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QSlider" name="amplitudeSlider">
       <property name="geometry">
        <rect>
         <x>17</x>
         <y>45</y>
         <width>131</width>
         <height>19</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QRadioButton" name="boatButton">
       <property name="geometry">
        <rect>
         <x>17</x>
         <y>175</y>
         <width>191</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Show Boat</string>
       </property>
      </widget>
      <zorder>label</zorder>
      <zorder>speedSlider</zorder>
      <zorder>energySlider</zorder>
      <zorder>label_2</zorder>
      <zorder>label_3</zorder>
      <zorder>amplitudeDisplay</zorder>
      <zorder>speedDisplay</zorder>
      <zorder>energyDisplay</zorder>
      <zorder>horizontalLayoutWidget</zorder>
      <zorder>amplitudeSlider</zorder>
      <zorder>boatButton</zorder>
     </widget>
    </item>
    <item>
     <widget class="GLWidget" name="widget" native="true">
      <zorder>groupBox</zorder>
      <zorder>groupBox</zorder>
      <zorder>groupBox</zorder>
      <zorder>groupBox</zorder>
      <zorder>groupBox</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
